All_Relationships/Experiment 1/job_sub.sh

```diff
@@ -0,0 +1,3 @@
+source set_env.sh
+cd All_Relationships
+bsub -o ~/sub-output.out -e ~/sub-error.out -q gpu -R "select[ngpus>0] rusage [ngpus_shared=1]" -n 1 python LSTM.py
```

Comment:
one nice trick for git diffs and readability is to split long shell commands onto multiple lines:

```sh
bsub \
  -o ~/sub-output.out \
  -e ~/sub-error.out -q gpu \
  -R "select[ngpus>0] rusage [ngpus_shared=1]" \
  -n 1 \
  python LSTM.py
```